README.md

```diff
@@ -54,7 +54,7 @@ articles:
   - title: How-to Cosmos Pt. 4- Building your first application with CosmJS
     date: Sunday, June 4
     time: 8
-    url: https://medium.com/lava-network/tutorial-how-to-cosmos-pt-4-building-your-first-application-with-cosmjs-87c5db9f3cdf
+    url: bit.ly/43wVQne

```

Comment:
You don't think that it's additional risk that the link conversion does not work.

Reply:
@coldice assured me it should work without any issues